Colours.m

```diff
@@ -46,12 +46,32 @@ @implementation NSColor (Colours)
 #pragma mark - Color from Hex
 + (instancetype)colorFromHexString:(NSString *)hexString
 {
-    unsigned rgbValue = 0;
+    unsigned long long rgbValue = 0;
+    int base = 0;
+    int mask = 0;
+
     hexString = [hexString stringByReplacingOccurrencesOfString:@"#" withString:@""];
     NSScanner *scanner = [NSScanner scannerWithString:hexString];
-    [scanner scanHexInt:&rgbValue];
-    
-    return [[self class] colorWithR:((rgbValue & 0xFF0000) >> 16) G:((rgbValue & 0xFF00) >> 8) B:(rgbValue & 0xFF) A:1.0];
+    [scanner scanHexLongLong:&rgbValue];
+
+    switch (hexString.length) {
+        case 3:
+            base = 8;
```

Comment:
function hasAnyCalc() {
        return hasCalc("-moz-") || hasCalc("-ms-") || hasCalc("-o-") || hasCalc("-webkit-") || hasCalc();
      }

      function calcHeight() {
        return window.innerHeight - {{#newScript}}190{{/newScript}}{{^newScript}}303{{/newScript}};
      }

      editor.setTheme("ace/theme/dawn");
      editor.getSession().setMode("ace/mode/javascript");

      {{#readOnly}}editor.setReadOnly(true);{{/readOnly}}
      {{^readOnly}}
      // TODO: Submit using AJAX
      $('#submit_code').click(function () {
        $('#source').val(editor.getValue());
        $('#code_form').submit();
      });
      {{/readOnly}}

      // Older browser JavaScript work-around for #136
      if (!hasAnyCalc()) {
        $("#editor").height(calcHeight());
        $(document).on("resize", function () {
          $("#editor").height(calcHeight);
        });
      }
    });

  })();
</script>





kincaid/

request_id=1&ios_base_sdk=9.3&platform=iPhone&app_name=26.iphone.com.tigervpns.hideme.ios.HideMe&u_so=p&js=afma-sdk-i-v7.6.0&ios_app_volume=1&ios_output_volume=0.344048&eid=318475625%2C46621136&ms=JWrkcPUl_IpumByXcxOud6yFysZvvAIBNHUclooCFb5hqW2yBQz5sjBIau1rDDJOWEQ37NGzhXiHutUuPswb4oQ-OwqwF3G-0WQwYIh2204cAo-

User-agent: *
Disallow: /admin/
Disallow: /mod/
Disallow: /flag/
Disallow: /vote/
Disallow: /auth/
Disallow: /github/
Disallow: /css/
Disallow: /dist/
Disallow: /fonts/
Disallow: /js/
Disallow: /redist/